app/models/payments/ChargeType.scala

```diff
@@ -198,6 +198,31 @@ case object PaymentOnAccountReturnCreditCharge extends ChargeType {
 case object PaymentOnAccountInstalments extends ChargeType {
   override val value: String = "VAT POA Instalments"
 }
+
+case object VatReturnPOALPI extends ChargeType {
+  override val value: String = "VAT Return POA LPI"
+}
+
+case object VatPOAInstalmentLPI extends ChargeType {
```

Comment:
POA Instalment charge isn't included in this subtask, it's just the other 5. Can you remove this and related parts for this charge. Ordinarily I would say leave it in but the related backend and stub data work don't support this so it can't be tested.